omod/src/main/java/org/openmrs/module/metadatasharing/web/controller/ImportController.java

```diff
@@ -566,8 +578,40 @@ public void assessItem(@ModelAttribute(ITEMS)
 		assessItemForm.setIndex(index);
 		assessItemForm.setImportType(importedItem.getImportType());
 		model.addAttribute(assessItemForm);
+		
+		XStream xstream= new XStream(new StaxDriver());
+		String xml1=xstream.toXML(importedItem.getIncoming());
+		String xml2=xstream.toXML(importedItem.getExisting());
+
+		String xmlIncomingString=formatXml(xml1);
+		String xmlExistingString=formatXml(xml2);
+
+		model.addAttribute("xmlExistingString",xmlExistingString);		
+		model.addAttribute("xmlIncomingString", xmlIncomingString);
+
 	}
 	
+	 public static String formatXml(String xml) {
+		   
+	      try {
+	         Transformer serializer = SAXTransformerFactory.newInstance().newTransformer();
+	         
+	         serializer.setOutputProperty(OutputKeys.INDENT, "yes");
+	         serializer.setOutputProperty("{http://xml.apache.org/xslt}indent-amount", "2");
+	         
+	         Source xmlSource = new SAXSource(new InputSource(
+	            new ByteArrayInputStream(xml.getBytes())));
+	         StreamResult res =  new StreamResult(new ByteArrayOutputStream());            
+	         
+	         serializer.transform(xmlSource, res);
+	         
+	         return new String(((ByteArrayOutputStream)res.getOutputStream()).toByteArray());
+	         
+	      } catch(Exception e) {
+	         return xml;
```

Comment:
Are you intentionally ignoring this exception?

Reply:
Let me handle it...Thanks